docs/scripting.md

```diff
@@ -655,7 +655,7 @@ module.exports = (robot) ->
     # Log commands
     robot.logger.info "#{context.response.message.user.name} asked me to #{context.response.message.text}"
     # Continue executing middleware
-    next(done)
+    next()
```

Comment:
Removed this because the "complex" example below shows how to properly wrap `done.